src/auth/reference_modules/oidc.py

```diff
@@ -57,7 +57,7 @@ def _parse_response(response):
 def _load_role_mappings(raw_role_mappings: str) -> dict:
     if raw_role_mappings:
         role_mapping = {}
-        raw_role_mappings = raw_role_mappings.split(";")
+        raw_role_mappings = raw_role_mappings.strip(" ;").split(";")
```

Comment:
Can you just give me an example here?

Reply:
This is just so when you have trailing ; it is ignored instead of throwing an error

Usually it would be `sso-role1:mg_role1; sso-role2:mg_role2`
Now it can also be `sso-role1:mg_role1; sso-role2:mg_role2;`

Docs already specify how it should be but one user mistakenly added the `;` at the end and I thought it wouldn't hurt to add this.

Reply:
I would rather split and then control the array

Reply:
Just to clarify why.
Your fix is very specific. You will have another user inputting "sso-role1:mg_role1;;sso-role2:mg_role2" and then we are back to the same problem. 
Split and then check the result. If empty remove, and throw if not compliant in a way we can't handle.